api/app/services/jwt_service.rb

```diff
@@ -1,6 +1,6 @@
 class JwtService
   ALGORITHM = "HS256"
-  ACCESS_TTL = 30.minutes
+  ACCESS_TTL = 1.days
```

Comment:
Access token TTL was increased to 1 day which lengthens exposure if a token is stolen. Revert to a short access TTL or make it configurable; for example use 30.minutes or read from ENV.